en/docs/develop/rest-apis.md

```diff
@@ -1,16 +1,4 @@
 # REST APIs
 
 This section provides information on REST APIs in WSO2 Identity Server
-including the sample requests and responses.
```

Comment:
Do we need this page? Since it doesn't have any content, shall we remove it? Or add the links to the child pages?

Reply:
This was required as certain other page was pointing to this section.